fortuna/src/config/run.rs

```diff
@@ -27,4 +31,16 @@ pub struct RunOptions {
     #[arg(long = "provider")]
     #[arg(env = "FORTUNA_PROVIDER")]
     pub provider: Address,
+
+    /// Path to a file containing a 20-byte (40 char) hex encoded Ethereum private key.
+    /// This key is required to submit transactions (such as registering with the contract).
+    #[arg(long = "private-key")]
```

Comment:
maybe `keeper-private-key`?
Also mention that this key should not have any specific privilege but just used for submitting transactions.